src/controllers/controller.cpp

```diff
@@ -1,9 +1,4 @@
-/**
-* @file controller.h
-* @author Sean Pappalardo spappalardo@mixxx.org
-* @date Sat Apr 30 2011
-* @brief Base class representing a physical (or software) controller.
-*/
+// Base class representing a physical (or software) controller.
```

Comment:
I intentionally did not do that, because that would be more than a simple regex and most likely require a lot of manual intervention. Besides, if it is a documentation comment on the class I'd expect it to be checked for currentness.
I'd rather have a person who knows the classes do that on the side when editing the class anyways. This PR only ensures that they don't have to deal with a lot of annoying surroundings.

Reply:
@Holzhaus

Reply:
@Holzhaus